distribution/feature-packs/server-feature-pack/src/main/resources/content/bin/migrate-domain-clustered.cli

```diff
@@ -540,4 +540,65 @@ if (outcome == success) of /socket-binding-group=$clusteredProfile-sockets/socke
   /profile=$clusteredProfile/socket-binding-group=$clusteredProfile-sockets/socket-binding=modcluster:undefine-attribute(name=port)
 end-if
 
-echo *** End Migration of /profile=$clusteredProfile ***
\ No newline at end of file
+# Migrate from 4.5.0 to 4.6.0
+if (outcome == failed) of /profile=$clusteredProfile/subsystem=microprofile-config-smallrye/:read-resource
+    try
+        echo Trying to add microprofile-config-smallrye extension
+        /extension=org.wildfly.extension.microprofile.config-smallrye/:add(module=org.wildfly.extension.microprofile.config-smallrye)
+        echo
+    catch
+        echo Wasn't able to add microprofile-config-smallrye extension, it should be already added by migrate-domain-standalone.cli
+        echo
+    end-try
+
+    echo Adding subsystem microprofile-config-smallrye
+    /profile=$clusteredProfile/subsystem=microprofile-config-smallrye/:add()
+    echo
+end-if
+
+if (outcome == failed) of /profile=$clusteredProfile/subsystem=microprofile-opentracing-smallrye/:read-resource
+    try
+        echo Trying to add microprofile-opentracing-smallrye extension
+        /extension=org.wildfly.extension.microprofile.opentracing-smallrye/:add(module=org.wildfly.extension.microprofile.opentracing-smallrye)
+        echo
+    catch
+        echo Wasn't able to add microprofile-opentracing-smallrye extension, it should be already added by migrate-domain-standalone.cli
+        echo
+    end-try
+
+    echo Adding subsystem microprofile-opentracing-smallrye
+    /profile=$clusteredProfile/subsystem=microprofile-opentracing-smallrye/:add()
+    echo
+end-if
+
+if (outcome == success) of /profile=$clusteredProfile/subsystem=elytron/http-authentication-factory=application-http-authentication/:read-resource
+    echo Removing application-http-authentication from elytron subsystem
+    /profile=$clusteredProfile/subsystem=elytron/http-authentication-factory=application-http-authentication:remove
+    echo
+end-if
+
+if (result != expression "${jboss.tx.node.id:1}") of /profile=$clusteredProfile/subsystem=transactions/:read-attribute(name=node-identifier)
+    echo Setting node-identifier attribute of core-environment element in transactions subsystem
+    /profile=$clusteredProfile/subsystem=transactions/:write-attribute(name=node-identifier,value=expression "${jboss.tx.node.id:1}")
```

Comment:
Use default value instead of `1` (dtto other cli scripts)

```suggestion
    /profile=$clusteredProfile/subsystem=transactions/:write-attribute(name=node-identifier,value=expression "${jboss.tx.node.id:${jboss.node.name:}}")
```